test/langtools/tools/javac/patterns/PrimitiveUnconditionallyExactTest.java

```diff
@@ -31,8 +31,7 @@
 import jdk.test.lib.compiler.InMemoryJavaCompiler;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.*;
 
 /*
  * @test
```

Comment:
The name of this test should be `T8357185` I think.

Reply:
The `T<number>` format is mostly frowned upon, and we mostly try to not use the convention for new tests. If we can have a good name, that's better.